doc/classes/Node.xml

```diff
@@ -239,6 +239,12 @@
 				This function ensures that the calling of this function will succeed, no matter whether it's being done from a thread or not. If called from a thread that is not allowed to call the function, the call will become deferred. Otherwise, the call will go through directly.
 			</description>
 		</method>
+		<method name="can_auto_translate" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if any text should automatically change to its translated version depending on the current locale. See also [member auto_translate_mode].
```

Comment:
This could actually go into detail by mentioning `atr` and `atr_n`, but as of writing I have no suggestions.